docs/getting-started/walk-through/advanced-hera-features.md

```diff
@@ -232,3 +234,62 @@ This feature can be enabled by setting the `experimental_feature` flag `script_a
 ```py
 global_config.experimental_features["script_annotations"] = True
 ```
+
+##### Outputs
+The annotations can also be used for outputs. An example can be seen [here](../../examples/workflows/script_annoations_outputs.md)
+
+Hera offers this for a simple hello world example:
+```python
+@script(outputs=Artifact(name="hello-art", path="/tmp/hello_world.txt"))
+def whalesay():
+   with open("/tmp/hello_world.txt", "w") as f:
+       f.write("hello world")
+```
+
+The new approach would allow us to avoid duplication of the path and result in a much more readable code.
+```python
+@script()
+def func() -> Annotated[str, Artifact(name="hello-art", path="/tmp/hello_world.txt")]:
+    return "hello world"
+```
+
+The idea is to save the returned value in a file according to this schema:
+`/hera/outputs/parameters/<name>`
+`/hera/outputs/artifacts/<name>`
+for easy access in subsequent Steps. The output is also exposed in the `outputs` section of the resulting yaml. 
+
+The item returned from the function can be of any serialisable pydantic type. That arbitrary type needs to be 
+`Annotated` with an `Artifact` or `Parameter` which we refer to with the “`OutputItem`” alias here. One of 
```

Comment:
```suggestion
`Annotated` with an `Artifact` or `Parameter` which we refer to with the `OutputItem` alias here. One of 
```

Reply:
Actually, reading it through I don't think the alias is helpful here, especially for `Tuple[Annotated[arbitrary_pydantic_type_a, OutputItem1], Annotated[arbitrary_pydantic_type_b, OutputItem2], ...]` - it starts to confuse what is the arbitrary type. Plus it's new terminology only used to introduce this feature. Can we replace "`OutputItem`" with "`Artifact`/`Parameter`" in the text, and choose one of the types for the examples (or show with both)?